### PR TITLE
add some usage examples

### PR DESCRIPTION
**Pull request by paed01**
- add query selectors to document
- example of capture navigation
- example capture DOM mutation

### CODE REVIEW

File: docs/Examples.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+});
+```
+
+## Capture DOM manipulation with mutation observer
+
+```javascript
+"use strict";
+
+const Browser = require("@expressen/tallahassee");
+const MutationObserver = require("@expressen/tallahassee/lib/MutationObserver");
```
Comment (jonaswalden) [inline docs/Examples.md:58]: This should also come prepacked on Window. It's even supported by IE11.

**Review by andolf — APPROVED** [review]

File: test/window-test.js [hunk 2]
```diff
@@ -34,6 +34,10 @@ describe("Window", () => {
     it("has FormData", () => {
       expect(window).to.have.property("FormData");
     });
+
+    it("has MutationObserver", () => {
+      expect(window).to.have.property("MutationObserver").that.equal(MutationObserver);
```
Comment (jonaswalden) [inline test/window-test.js:39]: isn't it overkill to check value? actual functionality is confirmed by other test

```suggestion
      expect(window).to.have.property("MutationObserver");
```

**Review by jonaswalden — APPROVED** [review]
👍 👍

File: test/window-test.js (hunk 2 above)
Comment (paed01) [reply test/window-test.js:39]: TDD forced me...

**Pull request merged**